.vsts-ci.yml

```diff
@@ -89,17 +89,29 @@ steps:
 
 # Publish VSTS artifact that the RoslynInsertionTool is able to find by its name.
 # The backing storage of the artifact is a file share.
+#
+# The bulk of the publish work is done by the Powershell task below. This exists to create
+# the proper entries in the build log so that the insertion tool can find the drop that 
+# needs to get published
 - task: PublishBuildArtifacts@1
   displayName: Publish Drop - Binaries
```

Comment:
Rename this to `Publish Drop  - DevDivPackages`, and the one below to just `Binaries`?